pom.xml

```diff
@@ -92,6 +92,19 @@
             <version>4.3.2</version>
         </dependency>
 
+        <!-- Swagger Documentation -->
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-swagger2</artifactId>
+            <version>2.9.2</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-swagger-ui</artifactId>
```

Comment:
Do we use this dependency?

Reply:
We use it [here](https://github.com/moltimate/moltimate-backend/pull/86/files#diff-2219e319a133ad07917715662738c7f4R25) and for the addition to [Application](https://github.com/moltimate/moltimate-backend/pull/86/files#diff-aaca4f42a170431e2736d3b2e26695cc) for Swagger-UI (it also pulls in the correct swagger components)